instaham/Gemfile

```diff
@@ -0,0 +1,67 @@
+source 'https://rubygems.org'
+
```

Comment:
Do the following and `rvm` will auto-select the right ruby version when you `cd` into a folder:

```rb
ruby '2.2.1'
```